Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -123,13 +123,17 @@ - (CGSize)intrinsicContentSize
 
 - (void)contextMenuInteraction:(UIContextMenuInteraction *)interaction willDisplayMenuForConfiguration:(UIContextMenuConfiguration *)configuration animator:(id<UIContextMenuInteractionAnimating>)animator
 {
+#if !PLATFORM(APPLETV)
```

Comment:
is this actually needed anymore now that there's the additional guard in PlatformEnableCocoa?

Reply:
No, you're right